admin_guide/scheduling/descheduler.adoc

```diff
@@ -0,0 +1,326 @@
+. Create the xref:admin-guide-descheduler-job[descheduler job configuration].
+. xref:admin-guide-descheduler-run[Run the descheduler job].
+
+
+[[admin-guide-descheduler-role]]
+== Creating a Cluster Role
+
+To configure the necessary permissions for the descheduler to work in a pod: 
+
+. Create a xref:../../architecture/additional_concepts/authorization.adoc#roles[cluster role] with the following rules:
++
+[source,yaml]
+----
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: descheduler-cluster-role
+rules:
+- apiGroups: [""]
+  resources: ["nodes"]
+  verbs: ["get", "watch", "list"] <1>
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "watch", "list", "delete"] <2>
+- apiGroups: [""]
+  resources: ["pods/eviction"] <3>
+  verbs: ["create"]
+----
+
+<1> Configures the role to allow viewing nodes.
+<2> Configures the role to allow viewing and deleting pods.
+<3> Allows a node to evict pods bound to itself.
+
+. Create the xref:../../admin_guide/service_accounts.adoc#admin-guide-service-accounts[service account] which will be used to run the job:
++
+----
+# oc create sa <file-name>.yaml -n kube-system
+----
++
+For example:
++
+----
+# oc create sa descheduler-sa.yaml -n kube-system
+----
+
+. Bind the cluster role to the service account:
++
+----
+# oc create clusterrolebinding descheduler-cluster-role-binding \
+    --clusterrole=<cluster-role-name> \
+    --serviceaccount=kube-system:<service-account-name>
+----
++
+For example:
++
+----
+# oc create clusterrolebinding descheduler-cluster-role-binding \
+    --clusterrole=descheduler-cluster-role \
+    --serviceaccount=kube-system:descheduler-sa
+----
+
+[[admin-guide-descheduler-policies]]
+== Creating Descheduler Policies
+
+The descheduler can be configured to remove pods from nodes that violate rules defined by _strategies_ in a YAML policy file. Include a path to the 
+policy file in the xref:admin-guide-descheduler-job[job specification] to apply the specific descheduling strategy.
+
+.Sample descheduler policy file
+[source,yaml]
+----
+apiVersion: "descheduler/v1alpha1"
+kind: "DeschedulerPolicy"
+strategies:
+  "RemoveDuplicates":
+     enabled: false
+  "LowNodeUtilization":
+     enabled: true
+     params:
+       nodeResourceUtilizationThresholds:
+         thresholds: <1>
+           "cpu" : 20
+           "memory": 20
+           "pods": 20
+         targetThresholds: <2>
+           "cpu" : 50
+           "memory": 50
+           "pods": 50
+         numberOfNodes: 3 <3>
+  "RemovePodsViolatingInterPodAntiAffinity": <1>
+     enabled: true
+----
+
+There are three default strategies that can be used with descheduler:
+
+* Remove duplicate pods (`RemoveDuplicates`)
+* Move pods to underutilized nodes (`LowNodeUtilization`) 
+* Remove pods that violate anti-affinity rules (`RemovePodsViolatingInterPodAntiAffinity`). 
+
+The parameters associated with these strategies can be configured and strategies can be disabled as needed.
```

Comment:
"You can configure and disable parameters associated with strategies as needed."